metrics/storage/webtool.sh

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Description of the test:
+# This test runs the 'web tooling benchmark'
+# https://github.com/v8/web-tooling-benchmark
+
+set -e
+
+# General env
+SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
+source "${SCRIPT_PATH}/../lib/common.bash"
+
+TEST_NAME="web-tooling"
+IMAGE="local-web-tooling"
+DOCKERFILE="${SCRIPT_PATH}/web-tooling_dockerfile/Dockerfile"
+CI_JOB=${CI_JOB:-""}
+configuration_file="/usr/share/defaults/kata-containers/configuration.toml"
+# Directory to run the test on
+# This is run inside of the container
+TESTDIR="${TESTDIR:-/testdir}"
+CMD="mkdir -p ${TESTDIR}; cd web-tooling-benchmark/ && node dist/cli.js"
+
+function main() {
+	# Check tools/commands dependencies
+	cmds=("docker")
+
+ 	init_env
+	check_cmds "${cmds[@]}"
+	check_dockerfiles_images "$IMAGE" "$DOCKERFILE"
+
+	metrics_json_init
+	local output=$(docker run --rm --runtime=$RUNTIME $IMAGE bash -c ''"$CMD"'')
+
+	# Save configuration
+	metrics_json_start_array
+
+	local acorn=$(echo "$output" | grep -w "acorn" | cut -d':' -f2 | cut -d' ' -f3-4)
+	local babel=$(echo "$output" | grep -w "babel" | head -1 | cut -d':' -f2 | cut -d' ' -f3-4)
```

Comment:
I'd be tempted to simplify this by creating a function that accepts `$output` and `$thing_to_find` and returns the final value. That will make this block less "dense" and make it easier to update when the format of `$output` changes.